map/framework.cpp

```diff
@@ -462,6 +465,17 @@ Framework::Framework()
   LOG(LINFO, ("Editor initialized"));
 
   m_trafficManager.SetCurrentDataVersion(m_storage.GetCurrentDataVersion());
+
+#ifdef OMIM_OS_ANDROID
+  ZipFileReader::UnzipFile(GetPlatform().ResourcesDir(),
```

Comment:
Good point about exceptions, handling added. Transliterators can be used in UI thread and we already do the same in the Storage constructor, for example:
`GetPlatform().GetReader(pathToCountriesFile)) `(on Android platform)